### PR TITLE
Add apple pay button

### PR DESCRIPTION
**Pull request by reastyn**
This will add Apple pay button to the payments

Issue: #1499

### CODE REVIEW

File: src/components/_payment.less [hunk 1]
```diff
@@ -25,3 +25,4 @@
 }
 
 @import "payment/_gpay.less";
+@import "payment/_apay.less";
```
Comment (shahednasser) [inline src/components/_payment.less:28]: Change the name of the file to `_apple-pay.less`

File: src/components/payment/_apay.less [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.payment-btn {
+  &.apay {
```
Comment (shahednasser) [inline src/components/payment/_apay.less:2]: change the name of the class to `.apple-pay`

File: website/src/data/buttons.json [hunk 3]
```diff
@@ -627,12 +627,20 @@
...
-        "children": [{
+        "children": [
+            {
                 "type": "gpay",
                 "classes": "gpay",
                 "label": "Pay with G Pay",
                 "text": "Pay with",
                 "usesColors": false
+            },
+            {
```
Comment (shahednasser) [inline website/src/data/buttons.json:638]: Make changes from `apay` to `apple-pay` and place `apple-pay` before `gpay`

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hello @reastyn great work! I just have some comments related to the naming of the class. It's best to make the name `apple-pay` instead of `apay` as it will be easier to remember when using it.

Also, in `buttons.json` you'll need to place `apple-pay` before `gpay` due to alphabetical order.

**Comment by reastyn** [comment]
Thank you for your feedback, I've fixed all the listed issues

**Review by shahednasser — CHANGES_REQUESTED** [review]
It seems that you forgot to run `npm run compile` and `npm run clean-css`, as the changes are not reflected in the CSS files. Please run these commands then commit the changes.

**Comment by reastyn** [comment]
You're right, hopefully now it is alright

**Comment by shahednasser** [comment]
Great thanks!

@all-contributors please add @reastyn for code

**Comment by reastyn** [comment]
Awesome, would you please put the `hacktoberfest-accepted` label to this pull request? Thank you!

**Pull request merged**